src/test/scala/com/microsoft/ml/spark/logging/LoggingTest.scala

```diff
@@ -0,0 +1,337 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.ml.spark.logging
+
+
+import com.microsoft.ml.spark.cognitive._
+import com.microsoft.ml.spark.cognitive.split1.{AnomalyKey, CognitiveKey, TextKey}
+import com.microsoft.ml.spark.core.test.base.TestBase
+import com.microsoft.ml.spark.stages.FixedMiniBatchTransformer
+import org.apache.commons.compress.utils.IOUtils
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
+import org.apache.spark.sql.functions.{col, collect_list, lit, struct}
+import org.apache.spark.sql.{DataFrame, Row}
+
+import java.net.URL
+
+class LoggingTest extends TestBase with TextKey with CognitiveKey with AnomalyKey{
+
+  import spark.implicits._
+
+  test("TextSentiment test"){
```

Comment:
how is this testing logging?

Reply:
Hi this is just a draft for test, please ignore it. Thanks still!